lib/driver.js

```diff
@@ -284,11 +283,6 @@ class AndroidDriver extends BaseDriver {
     if (this.isChromeSession) {
       // start a chromedriver session and proxy to it
       await this.startChromeSession();
-    } else {
-      if (this.opts.autoLaunch) {
```

Comment:
since `autoLaunch` defaults to true, we don't want to remove this block. We only want to remove behavior that would occur if `this.opts.autoLaunch` is *not* true.